apps/submission/io/pixel.py

```diff
@@ -0,0 +1,180 @@
+import pandas
+
+from django.utils.translation import ugettext as _
+
+from apps.core.models import OmicsUnit, Pixel, PixelSet
+from apps.data.models import Entry
+from ..exceptions import PixelSetParserError, PixelSetParserSaveError
+
+
+class PixelSetParser(object):
+
+    def __init__(self,
+                 pixelset_path,
+                 description=None,
+                 analysis=None,
+                 omics_unit_type=None,
+                 strain=None):
+
+        self.pixelset = None
+        self.pixelset_path = pixelset_path
+        self.description = description
+        self.analysis = analysis
+        self.omics_unit_type = omics_unit_type
+        self.strain = strain
+        self.pixels = None
+        self.db_pixels = {
+            'new': [],
+            'update': [],
+        }
```

Comment:
Why not using `dict(new=[], update=[])` here?

Reply:
Good question. I prefer the `dict()` syntax, but it appears that it is slower than the literal syntax `{}` [1]. In our case, I think it only has a minor impact, but we need to choose one and use it a along.

[1] https://doughellmann.com/blog/2012/11/12/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/